yamllintrc

```diff
@@ -0,0 +1,13 @@
+extends: default
+
+ignore:
+  - '**/hotsos/tests/unit/fake_data_root/**'
+  - '**/.tox/**'
+  - '**/examples/**'
+  - '**/.github/**'
+
+rules:
+  # 80 chars should be enough, but don't fail if a line is longer
```

Comment:
this should be `120` chars instead of `80`

Reply:
So i was thinking that maybe we should ignore this one completely and my reasoning is that having to wrap the search expressions over multiple lines makes it really easy to inadvertently insert an unexpectced whitespace char which could break the expr.

Reply:
You have a valid point, but I would argue that we should keep the line limit. IMO, if an expression is longer than 120 characters, it should have a valid justification and the warning should be suppressed only if expressing the check with a shorter expression is not possible. This would encourage refactoring the expressions. Lucky for us, the number of long expressions is rather small so I can go ahead and add the suppressions without having to comment on every single "expr" with the exception.

Reply:
Updated the MP: now long expressions are kept as-is and the warning is suppressed individually.

Reply:
Updated again, removed the line length check altogether.